app/forms/schools/add_participants/base_wizard.rb

```diff
@@ -464,27 +463,27 @@ def participant_cohort
 
       # NOTE: not preventing registration here just determining where to put the participant
       def cohort_to_place_participant
-        if transfer?
-          existing_participant_cohort || existing_participant_profile&.schedule&.cohort
-        elsif ect_participant? && induction_start_date.present?
-          Cohort.containing_date(induction_start_date).tap do |cohort|
-            return Cohort.current if cohort.blank? || cohort.npq_plus_one_or_earlier?
-          end
-        elsif Cohort.within_automatic_assignment_period?
-          # true from 1/9 to end of automatic assignment period
-          Cohort.current
-        elsif start_term == "summer"
-          Cohort.current
-        # we're in the registration window prior to 1/9
-        elsif start_term.in? %w[autumn spring]
-          # we're in the registration window prior to 1/9 and chose autumn or spring the following year
-          Cohort.next
-        else
-          # default to now - but should ask the start_term question if not already asked
-          Cohort.current
+        return cohort_for_transfer if transfer?
+        return cohort_for_ect_with_induction_start_date if ect_participant? && induction_start_date.present?
+        return Cohort.current if Cohort.within_automatic_assignment_period?
+
+        Cohort.next
```

Comment:
It looks like the default here has changed from `Cohort.current` to `Cohort.next`; is it the case that if the `start_term` is not `autumn` or `spring` then `Cohort.within_automatic_assignment_period?` holds `true`?

Reply:
if the `start_term` is not `autumn` or `spring` then `Cohort.within_automatic_assignment_period?` == false

Reply:
Actually the logic change this year is to place the participant in the next cohort if they are registered during summer term.